src/pubsub.js

```diff
@@ -113,6 +136,9 @@
             found = Boolean(messages.hasOwnProperty( topic ) && hasKeys(messages[topic]));
         }
 
+        if(!found)
```

Comment:
Please use the same brace style as the rest of the codebase. I'll try to update the eslint config to give better guidance.